findme-users/handler.py

```diff
@@ -53,10 +60,61 @@ def get_individual_user():
     return user_service.get_user(user_id)
 
 
+@app.get("/users/search")
+@tracer.capture_method
+@authorizer.requires_auth(app=app)
+def get_similar_users():
+    username = app.current_event.query_string_parameters.get("username")
+    return user_service.get_similar_users(username)
+
+
+@app.put("/users/<user_id>/follow")
+@tracer.capture_method
+@authorizer.requires_auth(app=app)
+def follow_user(user_id: Annotated[int, Path(lt=999)]):
+    requestee_id = app.context.get('claims').get('sub')
+    if '|' in user_id:
+        requestee_id = requestee_id.split("|")[1]
+    return follower_service.create_follower_request(requestee_id, user_id)
+
+
+@app.patch("/users/<requester_id>/follow")
+@tracer.capture_method
+@authorizer.requires_auth(app=app)
+def update_follow_user(requester_id: Annotated[int, Path(lt=999)]):
+    requestee_id = app.context.get('claims').get('sub')
+    if '|' in requestee_id:
+        requestee_id = requestee_id.split("|")[1]
+    action = app.current_event.query_string_parameters.get("action")
+    if action == "accept":
+        return follower_service.accept_follow_request(requester_id, requestee_id)
+    if action == "declined":
+        return follower_service.create_follower_request(requester_id, requestee_id)
+    else:
+        raise BadRequestError(f"Action {action} does not exist, please provide a valid value (accept/decline)")
```

Comment:
in the code above you check for the action 'declined' but provide 'decline' as valid value

Reply:
fixed in 37c722e, please confirm